README.rst

```diff
@@ -17,7 +17,7 @@ packages. For example:
 
    $ virtualenv venv
    $ source venv/bin/activate
-   $ pip install -U pip
+   $ python -m pip install --upgrade pip
```

Comment:
Wondering why you needed to change this

Reply:
was a while ago so i can't really remember. I think it was possibly an issue on my system where `pip` pointed to system level pip, so we use `python -m pip` to use specifically venvs `pip`. FWIW they're functionally equivalent in this case.